pkgs/top-level/all-packages.nix

```diff
@@ -25813,6 +25813,8 @@ with pkgs;
 
   ## DEPRECATED, will be removed in a future release
 
+  eww-wayland = lib.warn "eww now can build for X11 and wayland simultaneously, so `eww-wayland` is deprecated, use the normal `eww` package instead." (callPackage ../applications/window-managers/eww {});
```

Comment:
This must be in aliases.nix next to this file

Reply:
Did not notice that thanks

Reply:
fixed